text/0001-telemetry-without-manual-instrumentation.md

```diff
@@ -91,7 +91,7 @@ There are some languages that will have OpenTelemetry support before they have D
 
 ### Governance of the auto-instrumentation libraries
 
-The maintainers for each language `foo` will retain their Approver/Maintainer status and privileges for the `auto-instr-foo` repositories.
+Each `auto-instr-foo` repository must have at least one [Maintainer](https://github.com/open-telemetry/community/blob/master/community-membership.md#maintainer) in common with the main `opentelemetry-foo` language repository. There are no other requirements or constraints about the set of maintainers/approvers for the main language repository and the respective auto-instrumentation repository.
```

Comment:
Also mentioned clearly that we should support having maintainers of auto that are not maintainers of the language and vice-versa.

Reply:
Sounds good. I made this explicit, @bogdandrutu, and will merge.